<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>plugin_selector</class>
 <widget class="QFrame" name="plugin_selector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>605</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>8</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>190</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>190</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_template_plugins_selection">
         <property name="minimumSize">
          <size>
           <width>210</width>
           <height>0</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="m_hovered_pushButton" name="pushButton_remove_template">
         <property name="minimumSize">
          <size>
           <width>21</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>21</width>
           <height>21</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:checked{ border: none;  }
QPushButton:!checked{  border: none;  } 
QPushButton:clicked{ border: none;}               
QPushButton:pressed { white; border: none; }
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_plugin_search">
         <property name="minimumSize">
          <size>
           <width>220</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>SegoeUI</family>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    background-repeat: no-repeat;
    background-position: left;
    font-family: SegoeUI;
    font-size: 12px;
    border-width: 2px;
    padding: 0 0 0 20; /* left padding (last number) must be more than the icon's width */
}</string>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>Plugin Search</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page_all_plugins">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>2</number>
           </property>
           <item>
            <widget class="QTableWidget" name="tableWidget_parent_list">
             <property name="minimumSize">
              <size>
               <width>190</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>190</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QTableView {
/*selection-background-color: #87CEFF;*/
selection-background-color: #0069EB;
selection-color:white;
 }






QTableView::item:selected:!active {
    color:white;
    
   /*background-color:  #0069EB;*/


}




 QTableView::item:selected:!active {
                                 color:white;
                                 background-color:  #0069EB;
                                  }


                                 QTableView {
                                 selection-background-color: #0069D7;
                                    }
    

                                 QTableView::item:selected:!active {
                                 color:white;
                                 background-color:  #0069EB;
                                 }

QScrollBar{
		width:9px;
}</string>
             </property>
             <property name="sizeAdjustPolicy">
              <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="showGrid">
              <bool>false</bool>
             </property>
             <property name="gridStyle">
              <enum>Qt::NoPen</enum>
             </property>
             <attribute name="horizontalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Translated Name</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Original Name</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <widget class="QTreeWidget" name="treeWidget_plugins">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QTreeWidget {

/*selection-background-color: #87CEFF;*/
 /*selection-background-color: #0069EB; */
selection-color:white;
 }

QTreeWidget::item{ 
	 height: 26px; 
 }


QTreeWidget::item:selected:!active {
    color:white;
    background-color:  #0069EB;
}

 
                                QTreeWidget {
                                 selection-background-color: #0069D7;
                                    } 


QScrollBar{
		width:9px;
}

QTreeWidget::indicator:checked {background-color:#777777;border: 1px solid #d6d6d6;}
QTreeWidget::indicator:unchecked {background-color:#f1f1f1;border: 1px solid #777777;}






QTreeWidget::indicator {
width: 13px;
height: 13px;
}


</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="rootIsDecorated">
              <bool>true</bool>
             </property>
             <property name="animated">
              <bool>true</bool>
             </property>
             <column>
              <property name="text">
               <string notr="true">Plugins</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Plugin Original Name</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string/>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Export</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>M</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>i</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>W</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>A</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Blank</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QCheckBox" name="checkBox_save_template">
         <property name="minimumSize">
          <size>
           <width>196</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>190</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string> Save Template</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_save_template">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <italic>false</italic>
          </font>
         </property>
         <property name="placeholderText">
          <string>Template Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_add_template">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>23</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	margin: 1px;
     border-color: #7f7f7f;
     border-style: solid;
     border-radius: 1px;
     border-width: 1px;
      }
QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #FFFFFF, stop: 1 #FFFFFF);
                border-color: #000000;
}

QPushButton:checked {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #FFFFFF, stop: 1 #FFFFFF);
                border-color: #484848;
}</string>
         </property>
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>m_hovered_pushButton</class>
   <extends>QPushButton</extends>
   <header location="global">ui_custom_classes/m_hovered_pushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
